sticht/slack.py

```diff
@@ -52,6 +52,14 @@ def log_error(msg):
         log.error(msg)
 
 
+class Emoji(str):
```

Comment:
just moved this out of the slo file since it didn't really seem like it fit in as a class in there